pandas/core/indexes/interval.py

```diff
@@ -457,7 +461,8 @@ def from_breaks(cls, breaks, closed='right', name=None, copy=False,
     def from_arrays(cls, left, right, closed='right', name=None, copy=False,
                     dtype=None):
         """
-        Construct an IntervalIndex from a a left and right array
+        Construct an IntervalIndex from a given element in a left
+        and right array.
```

Comment:
Can you try to keep this in a single line? I also don't fully understand the "from a given element in ", so would maybe remove that part